MicroPython/src/common/config.py

```diff
@@ -28,19 +24,15 @@
 DEFAULT_WIFI_TIMEOUT = 5000
 DEFAULT_MQTT_PORT = 1883
 DEFAULT_MQTT_PORT_SSL = 8883
-DEFAULT_MQTT_TIMEOUT = 4000
+DEFAULT_MQTT_TIMEOUT = 400
```

Comment:
Is any reason for this change?

Reply:
4000 seconds for timeout is in my opinion too many.